### PR TITLE
(feature): login and signup

### PR DESCRIPTION
**Pull request by Ethical-Ralph**
All conflict fixed
this is a major update that include changes to to the app.js, update to render error in views during authentication process.
it also redirect a dashboard that includes the token as query after successfull login
merge and test

### CODE REVIEW

File: controllers/usercontroller.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable */
```
Comment (tonymj76) [inline controllers/usercontroller.js]: please don't disable eslint try to fix the errors for our code should be uniform

File: routes/newroutes.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
```
Comment (tonymj76) [inline routes/newroutes.js]: also here

**Review by tonymj76 — CHANGES_REQUESTED** [review]
Nice work fix the merge conflict and some of the comment

**Comment by Ethical-Ralph** [comment]
Okay all done now merge and test

**Pull request merged**